app/code/Codifi/CustomerRequest/Controller/Request/Save.php

```diff
@@ -0,0 +1,162 @@
+     * @param StoreManagerInterface $storeManager
+     * @param Escaper $escaper
+     */
+    public function __construct(
+        Context $context,
+        CustomerNoteFactory $customerNoteFactory,
+        NoteRepository $customerNoteRepository,
+        TransportBuilder $transportBuilder,
+        StoreManagerInterface $storeManager,
+        Escaper $escaper
+    ) {
+        $this->customerNoteFactory = $customerNoteFactory;
+        $this->customerNoteRepository = $customerNoteRepository;
+        $this->transportBuilder = $transportBuilder;
+        $this->storeManager = $storeManager;
+        $this->escaper = $escaper;
+        parent::__construct($context);
+    }
+
+    /**
+     * Execute function
+     *
+     * @return ResultInterface
+     * @throws LocalizedException
+     * @throws MailException
+     * @throws NoSuchEntityException
+     */
+    public function execute(): ResultInterface
+    {
+        $request = $this->getRequest();
+        $customerId = $request->getParam('customer_id');
+        $customerEmail = $request->getParam('email_address');
+        $message = $request->getParam('customer_messsage');
+        $customerName = $request->getParam('customer_name');
+
+        if ($customerId && $customerEmail && $message && $customerName) {
+
+            $store = $this->storeManager->getStore();
+            $storeName = $store->getFrontendName();
+
+            $note = "Customer sent request from $storeName";
+
+            $sender = [
+                'name' => $this->escaper->escapeHtml($customerName),
+                'email' => $this->escaper->escapeHtml($customerEmail),
+            ];
+
+            $transport = $this->transportBuilder
+                ->setTemplateIdentifier('email_request_template')
+                ->setTemplateOptions(
+                    [
+                        'area' => Area::AREA_FRONTEND,
+                        'store' => Store::DEFAULT_STORE_ID,
+                    ]
+                )
+                ->setTemplateVars([
+                    'customerNameVar' => $customerName,
+                    'customerEmailVar' => $customerEmail,
+                    'messageVar' => $message,
+                ])
+                ->setFromByScope($sender)
+                ->addTo('support@example.com')
+                ->getTransport();
+            $transport->sendMessage();
+
+            $customerNoteModel = $this->customerNoteFactory->create();
+
+            try {
+                $customerNoteModel->setData([
+                    'customer_id' => $customerId,
+                    'note' => $note,
+                    'autocomplete' => 1
+                ]);
+                $this->customerNoteRepository->save($customerNoteModel);
+                $this->messageManager->addSuccessMessage(Config::SUCCESS_MESSAGE);
+            } catch (LocalizedException $exception) {
+                $this->messageManager->addErrorMessage(Config::ERROR_MESSAGE);
```

Comment:
If you try to catch exception for specific functionality, you should handle it appropriately (log it - the best way, or show this exception message - in this case).